README.md

```diff
@@ -62,11 +56,19 @@ shows you how to use a variety of Google Cloud Platform features.
 
 View the [tutorial][bookshelf_docs] or the [source code][bookshelf_code].
 
-[bookshelf_docs]: https://cloud.google.com/nodejs/getting-started/tutorial-app
-[bookshelf_code]: https://github.com/GoogleCloudPlatform/nodejs-getting-started
-
 ## Contributing
 
 Contributions welcome! See the [Contributing Guide][contrib].
 
-[contrib]: CONTRIBUTING.md
+[slack_badge]: https://img.shields.io/badge/slack-Google%20Cloud%20Platform-E01563.svg
```

Comment:
For now, leave this misc information as is in the readme and just have the ts build step added.

Reply:
Do you mean just the URLs I moved to the bottom of the page, or all of the changes I'm suggesting?

Reply:
I moved the URLs back to where they were